

Greg's tree changes the pci_save_state/pci_restore_state API.  Whoever merges
second needs this patch.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/net/ne2k-pci.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff -puN drivers/net/ne2k-pci.c~ne2k-pci-pci-build-fix drivers/net/ne2k-pci.c
--- 25/drivers/net/ne2k-pci.c~ne2k-pci-pci-build-fix	2004-10-01 21:09:23.642041600 -0700
+++ 25-akpm/drivers/net/ne2k-pci.c	2004-10-01 21:09:40.578466872 -0700
@@ -659,7 +659,7 @@ static int ne2k_pci_suspend (struct pci_
 	struct net_device *dev = pci_get_drvdata (pdev);
 
 	netif_device_detach(dev);
-	pci_save_state(pdev, pdev->saved_config_space);
+	pci_save_state(pdev);
 	pci_set_power_state(pdev, state);
 
 	return 0;
@@ -670,7 +670,7 @@ static int ne2k_pci_resume (struct pci_d
 	struct net_device *dev = pci_get_drvdata (pdev);
 
 	pci_set_power_state(pdev, 0);
-	pci_restore_state(pdev, pdev->saved_config_space);
+	pci_restore_state(pdev);
 	NS8390_init(dev, 1);
 	netif_device_attach(dev);
 
_
